pkg/snapshot/snapshot.go

```diff
@@ -0,0 +1,141 @@
+package snapshot
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/lightsail"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	defaultSnapshotSuffix = "auto-snapshot"
+)
+
+var (
+	defaultRetention = 10 * 25 * time.Hour
```

Comment:
default retention is 10 days and 10 hours that's a weird duration 🤣. I'm fine with it. 😉

Reply:
maybe 1337 hours would be better 🤔